### PR TITLE
submit a vulnerability

### PR DESCRIPTION
**Pull request by jesuscript**

### CODE REVIEW

File: client/autoform.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+AutoForm.hooks({
+  submitVulnerabilityForm: {
+    after: {
+      submitVulnerability: function(err, res){
+        if(err){
+          UI.setTextFeedback("Error","Vulnerability submission failed", "alert-danger");
+        }else{
+          Router.go("/");
+          UI.setTextFeedback("Thank you!", "Vunlnerability has been submitted",{
```
Comment (Gustav-Simonsson) [inline client/autoform.js]: typo

File: admin/config.js [hunk 2]
```diff
@@ -7,6 +7,21 @@ AdminConfig = {
...
+        label: "Accepted",
+        name: "accepted"
+      },{
+        label: "User",
+        name: "user",
+        collection: "Users"
+      }, {
+        label: "Title",
+        name: "title"
+      }]
```
Comment (Gustav-Simonsson) [inline admin/config.js]: Should also be a 'description' column?

Comment (jesuscript) [inline admin/config.js]: this defines only the fields to display in the "index" table. Descriptions could be long and as such would bloat the table. You can still them if you navigate to individual submissions

Comment (Gustav-Simonsson) [inline admin/config.js]: ok cool

**Pull request merged**